ui/less/containers.less

```diff
@@ -170,6 +170,11 @@
   }
 }
 
+/* Buttons hide certain items if they are found inside the control panel */
+.shaka-controls-button-panel .display-none-if-on-control {
```

Comment:
This class name feels weird to me.  I think "shaka-overflow-menu-only" might be a better name, but I'd be happy to debate it if you disagree.

As the saying goes, there's really only one hard problem in computer science: naming things, and off-by-one errors.  :grin:

Reply:
I am ashamed to admit how much time it took me to understand that saying the first time I read it. 

Oh well, I changed it now, let us see what other weird variable names I come up with next. 😄